Mapped additional properties to AccessTokenResponse and User

smanabat opened the pull request.

Inumedia commented:
Hi @smanabat, I'm not sure what the intention is with the auth token / new client secrets. Could you like to some change in the Slack official API or clarify?

Also, as I've mentioned on the other pull requests, I likely won't have time to fully review this / merge it in until this weekend, if not later.

smanabat commented:
Hi @Inumedia, I've added 2 new properties that were not mapped previously, bot_user_id and bot_access_token, that is returned from the AccessTokenResponse. 

The AuthCode, ClientId, and ClientSecrets in the config are used for the Integration test which I've added to test the mapping.

Check out "Bot user access tokens" on the [documentation](https://api.slack.com/docs/oauth) for the 2 properties I've included.
